tests/map.spec.js

```diff
@@ -166,3 +230,110 @@ test('test getTraversableCells', t => {
 
   t.same(map.getTraversableCells(cpt, smallMap), expected);
 });
+
+
+test('test init2dArray', t => {
+  t.plan(2);
+
+  let width = 5;
+  let height = 3;
+  let defaultVal = 1;
+  let expected = [
+    [1, 1, 1],
+    [1, 1, 1],
+    [1, 1, 1],
+    [1, 1, 1],
+    [1, 1, 1],
+  ];
+  t.same(map.init2dArray(width, height, defaultVal), expected);
+
+  width = 3;
+  height = 3;
+  defaultVal = 55;
+  expected = [
+    [55, 55, 55],
+    [55, 55, 55],
+    [55, 55, 55],
+  ];
+  t.same(map.init2dArray(width, height, defaultVal), expected);
+});
+
+
+test('test multiplyCorrespondingElementsAndSum', t => {
+  t.plan(1);
+
+  const m1 = [
+    [1, 2, 3],
+    [4, 5, 6],
+    [7, 8, 9],
+  ];
+
+  const m2 = [
+    [9, 8, 7],
+    [6, 5, 4],
+    [3, 2, 1],
+  ];
+
+  const expected = 165;
+
+  t.is(map.multiplyCorrespondingElementsAndSum(m1, m2), expected);
+});
+
+test('test convolve', t => {
+  t.plan(3);
+
+  let m = [
+    [1, 2, 3],
+    [4, 5, 6],
+    [7, 8, 9],
+  ];
+  let k = [
+    [1],
+  ];
+  let expected = [
+    [1, 2, 3],
+    [4, 5, 6],
+    [7, 8, 9],
+  ];
+  t.same(map.convolve(m, k), expected);
+
+  m = [
+    [1, 2, 3],
+    [4, 5, 6],
+    [7, 8, 9],
+  ];
+  k = [
+    [2],
+  ];
+  /* eslint-disable array-bracket-spacing*/
+  /* eslint-disable no-multi-spaces*/
+  expected = [
+    [ 2,  4,  6],
+    [ 8, 10, 12],
+    [14, 16, 18],
+  ];
+  /* eslint-enable array-bracket-spacing*/
+  /* eslint-enable no-multi-spaces*/
+  t.same(map.convolve(m, k), expected);
+
+  m = [
+    [1, 2, 3, 4],
+    [5, 6, 7, 8],
+    [9, 0, 1, 2],
+  ];
+  k = [
+    [1, 2, 3],
+    [3, 4, 5],
+    [6, 7, 8],
+  ];
+  /* eslint-disable array-bracket-spacing*/
+  /* eslint-disable no-multi-spaces*/
```

Comment:
You could reduce lines by surrounding all your tests in eslint disable for these two. Also, this can be one line i believe (not sure though):

`eslint-disable rule1 rule2`